storm-core/src/jvm/backtype/storm/utils/Utils.java

```diff
@@ -306,7 +327,30 @@ public static ComponentCommon getComponentCommon(StormTopology topology, String
         }
         throw new IllegalArgumentException("Could not find component with id " + id);
     }
-    
+
+    public static List<String> getStrings(final Object o) {
+        if (o == null) {
+            return Collections.emptyList();
```

Comment:
All of the other lists here are mutable, but this isn't. 
I'd rather see a new ArrayList, since it's a public method.